request.go

```diff
@@ -73,6 +81,12 @@ func (client *Client) doJsonRequest(method, api string,
 	}
 	defer resp.Body.Close()
 
+	if err := client.getRateLimit(resp); err != nil {
+		if err.(*RateLimit).Remaining == 0 {
```

Comment:
Would be 🏆 if we could back start backing off, if we have _X-RateLimit-Reset_ we know how long we'd have to wait for, right?